tests/test_fields.py

```diff
@@ -104,7 +104,7 @@ def test_fields_filtering(self):
         assert len(res['data']) == 1
         assert res['recordsTotal'] == '52'
         assert res['recordsFiltered'] == '1'
-        assert res['data'][0]['1'] == 'User 51'
```

Comment:
This should be an integer, as DataTables is using it as array.
See the first example at https://datatables.net/manual/server-side